examples/desktop/events/resilient_monitor.c

```diff
@@ -116,11 +116,17 @@ int main(int argc, char *argv[]) {
       // Body: %s\n", message->error_response);
       break;
     }
-    case ATCLIENT_MONITOR_ERROR_PARSE: {
+    case ATCLIENT_MONITOR_ERROR_PARSE_NOTIFICATION: {
       atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "Message type: ATCLIENT_MONITOR_ERROR_PARSE\n");
+      tries++;
+      break;
+    }
+    case ATCLIENT_MONITOR_ERROR_DECRYPT_NOTIFICATION: {
+      atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "Message type: ATCLIENT_MONITOR_ERROR_DECRYPT\n");
+      tries++;
       break;
     }
-    case ATCLIENT_MONITOR_ERROR_READ: {
+    case ATCLIENT_MONITOR_EMPTY_READ: {
       atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "Message type: ATCLIENT_MONITOR_ERROR_READ\n");
```

Comment:
Should be changed to:

`atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_DEBUG, "Message type: ATCLIENT_MONITOR_EMPTY_READ\n");`

Reply:
Thanks, fixed in https://github.com/atsign-foundation/at_c/pull/306/commits/6cf3551df955b15852b43a3d74f72a459fa50833